client/my-sites/plugins/plugins-list/plugins-list-dataviews.tsx

```diff
@@ -107,58 +100,12 @@ export default function PluginsListDataViews( {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [ pluginSlug ] );
 
-	const header = (
-		<>
```

Comment:
We could probably hide this when the preview is open or the filter is applied instead of removing this.

Reply:
I've reverted the changes, as discussed above.